src/webots/vrml/WbProtoManager.cpp

```diff
@@ -144,9 +144,8 @@ WbProtoModel *WbProtoManager::findModel(const QString &modelName, const QString
     bool foundProtoVersion = false;
     const WbVersion protoVersion = checkProtoVersion(parentFilePath, &foundProtoVersion);
     if (foundProtoVersion && protoVersion < WbVersion(2022, 1, 0)) {
-      const QString backwardsCompatibilityMessage =
-        tr("Please adapt your project to R2023a following these instructions: "
-           "https://github.com/cyberbotics/webots/wiki/How-to-adapt-your-world-or-PROTO-to-Webots-R2022b");
+      const QString backwardsCompatibilityMessage = tr("Please adapt your project to R2023a following these instructions: "
```

Comment:
This is not an issue with this PR, but I'm not sure it is good in R2023b to suggest to adapt the project to the old R2023a version.

Reply:
I agree. I will link the generic upgrade page because they will have to go through all the successive updates.